porting-quick.md

```diff
@@ -39,15 +39,15 @@ Add the target description to ```mbed-os\targets\targets.json```
 
 ### Bootstrap code
 
-There are a number of files needed before a first successful compile. CMSIS-CORE files are needed for start-up and peripheral memory addresses as well as the linker scripts for ARM, IAR and GCC toolchains. These files are usually in the ```mbed-os\targets\TARGET_VENDOR\TARGET_MCU_FAMILY\TARGET_MCUNAME\device``` directory.
+You need a number of files to first successfully compile. You need CMSIS-CORE files for startup and peripheral memory addresses as well as the linker scripts for ARM, IAR and GCC toolchains. These files are usually in the ```mbed-os\targets\TARGET_VENDOR\TARGET_MCU_FAMILY\TARGET_MCUNAME\device``` directory.
 
-If the microcontroller has a SYS_TICK the RTOS will configure this, so only a few macros are needed in ```mbed-os\targets\TARGET_VENDOR\mbed-rtx.h```.
+If the microcontroller has a SYS_TICK, the RTOS will configure this, so only a few macros are needed in ```mbed-os\targets\TARGET_VENDOR\mbed-rtx.h```.
 
-Some extensions to CMSIS-CORE are also required for dynamic vector relocation. ```mbed-os\targets\TARGET_VENDOR\TARGET_MCUNAME\cmsis.h``` is the entry point to the target for mbed OS software. This will include the vector relocation additions and device-specific headers that include CMSIS-CORE. A relocation routine is needed in ```mbed-os\targets\TARGET_VENDOR\TARGET_MCUNAME\cmsis_nvic.c and .h```
+Dynamic vector relocation also requires some extensions to CMSIS-CORE. ```mbed-os\targets\TARGET_VENDOR\TARGET_MCUNAME\cmsis.h``` is the entry point to the target for mbed OS software. This will include the vector relocation additions and device-specific headers that include CMSIS-CORE. A relocation routine is needed in ```mbed-os\targets\TARGET_VENDOR\TARGET_MCUNAME\cmsis_nvic.c and .h```
 
-Now verify your target is identified by using ```mbed compile -m MCU_NAME -t <toolchain>```. This should compile.
+Now verify your target is identified by using ```mbed compile -m MCU_NAME -t <toolchain>```. This compiles.
```

Comment:
This paragraph and the one below seem to be steps, and the paragraph below even starts with "The next step", but the first three paragraphs in the section seem to be background information. Is there a way to fix the flow of this?